deployment/live/gcp/static-ct-staging/logs/arche2025h1/README.md

```diff
@@ -0,0 +1,62 @@
+# Arche2025h1 staging log
+
+## Overview
+
+This config uses the [gcp/tesseract/conformance](/deployment/modules/gcp/tesseract/conformance) module to
+define a Tesseract log Cloud Run, backed by Trillian Tessera and preloaded with argon2025h1 entries.
```

Comment:
```suggestion
define a TesseraCT log Cloud Run, backed by Trillian Tessera and preloaded with argon2025h1 entries.
```

Reply:
Some word is missing between "TesseraCT log" and "Cloud Run". Is it `on` or `using` or something else?